test/e2e/specs/editor/blocks/quote.spec.js

```diff
@@ -3,29 +3,310 @@
  */
 const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
 
-test.describe( 'adding a quote', () => {
+test.describe( 'Quote', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
```

Comment:
We should probably clean up after all tests are done.

```js
test.afterAll( async ( { requestUtils } ) => {
  await requestUtils.deleteAllPosts();
} );
```

Reply:
Done.
By the way, I think that once #46459 is merged, The clean-up might no longer be needed 🤔

Reply:
Thanks. That would be great. We can remove post-cleanup methods after that is confirmed in #46459.